packages/components/src/DateTimePickers/PickerAction/PickerAction.component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import theme from './PickerAction.scss';
+
+function PickerAction(props) {
+	const className = classNames(
+		theme.action,
+		props.isSelected ? theme.selected : undefined,
+		props.className,
+	);
+
+	return (
+		<button
+			className={className}
```

Comment:
Still KO. From my previous review : `You forgot to spread the rest of props So in month picker you pass an aria-label but is not reflected in the button`